src/screens/docs/getting-started.md

```diff
@@ -8,7 +8,27 @@ In this guide, we’ll show you how to get started with Victory and walk you thr
 
 ### 1. Set up a basic React project
 
-You can do this on your own if you'd like, or you can clone down [this one that we've created for you](https://github.com/FormidableLabs/victory-tutorial/tree/260e1fddde48532707855177c0ecbc674b184823). If you're cloning down our example, remember to run an ```npm install``` to install all necessary dependencies. Once you've done this, you can run the webpack server with the command ```npm start```, and the project will render at ```localhost:8080```. All modifications will take place in your client.js file.
+You can do this on your own if you'd like, or you can clone down [this one that we've created for you](https://github.com/FormidableLabs/victory-tutorial), and replace the existing code in the ```client.js``` file with:
```

Comment:
I might explain how to clone. Just list the basic steps.
1. clone...
2. cd...
3. npm install...